### PR TITLE
Autogenerated avatars for users w/o image field

### LINKED ISSUE #333: Autogenerated avatars for users w/o image field

### PR DESCRIPTION
**Pull request by DenisVorop**
## PR includes
- [x] Feature

## Related issues
Resolve #333 

## QA Instructions, Screenshots, Reco

Снимок экрана 2023-08-11 в 07 13 09

### CODE REVIEW

File: src/components/Gravatar.tsx [hunk 1]
```diff
@@ -93,17 +109,23 @@ export const Gravatar = ({
...
-            onClick={onClick}
-            onError={onLoadError}
-        />
+        <>
+            {!isError ? (
+                <StyledImage
+                    visible={visible}
+                    ref={imgRef}
+                    alt={`Gravatar for ${formattedEmail}`}
+                    src="/anonymous.png"
```
Comment (awinogradov) [inline src/components/Gravatar.tsx:118]: I think you can remove it

**Comment by DenisVorop** [comment]
Снимок экрана 2023-08-14 в 12 25 44
Снимок экрана 2023-08-14 в 12 30 08

**Comment by DenisVorop** [comment]
Снимок экрана 2023-08-14 в 12 25 44
Снимок экрана 2023-08-14 в 12 30 08

File: src/components/Gravatar.tsx [hunk 2]
```diff
@@ -1,47 +1,70 @@
...
+    letter-spacing: -1.5px;
+    font-weight: 600;
+    user-select: none;
 `;
 
-export const Gravatar: React.FC<GravatarProps> = ({
+const getInitials = (fullName?: string | null) => {
+    if (!fullName) return '';
+    const arr = fullName.split(' ');
+    return `${arr.at(-2)?.[0] || ''} ${arr.at(-1)?.[0] || ''}`;
```
Comment (awinogradov) [inline src/components/Gravatar.tsx:52]: `arr[0]?.[0]`

**Review by awinogradov — CHANGES_REQUESTED** [review]
image

Must be removed

File: src/components/UserPic.tsx [hunk 3]
```diff
@@ -22,17 +23,11 @@ export const UserPic: React.FC<UserPicProps> = ({ src, email, size = 32, ...prop
 
     const onLoadError: React.ReactEventHandler<HTMLImageElement> = useCallback(({ currentTarget }) => {
         currentTarget.onerror = null;
-        currentTarget.src = '/anonymous.png';
```
Comment (awinogradov) [inline src/components/UserPic.tsx:25]: Why here we do nothing on error, but [here](https://github.com/taskany-inc/bricks/pull/334/files#diff-fc4e5466b03ed2b606adb7ab361ba653733fa7f0580d9c97b73126580d0dbb30R93) we are using Circle as fallback?

**Review by awinogradov — CHANGES_REQUESTED** [review]

**Review by 9teen90nine — APPROVED** [review]

**Pull request merged**